stonesoup/tracker/pointprocess.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+from .base import Tracker
+from ..base import Property
+from ..reader import DetectionReader
+from ..types.state import TaggedWeightedGaussianState
+from ..types.mixture import GaussianMixture
+from ..types.track import Track
+from ..updater import Updater
+from ..hypothesiser.gaussianmixture import GaussianMixtureHypothesiser
+from ..mixturereducer.gaussianmixture import GaussianMixtureReducer
+from ..buffered_generator import BufferedGenerator
+
+
+class PointProcessMultiTargetTracker(Tracker):
+    """
+    Base class for Gaussian Mixture (GM) style implementations of
+    point process derived filters
+    """
+    detector = Property(
+        DetectionReader,
+        default=None,
+        doc="Detector used to generate detection objects.")
+    updater = Property(
+        Updater,
+        default=None,
+        doc="Updater used to update the objects to their new state.")
+    gaussian_mixture = Property(
+        GaussianMixture,
+        default=None,
+        doc="""Gaussian Mixture modelling the
```

Comment:
The format of these documentation strings causes some strange formatting in the resulting documentation. Documentation for gaussian_mixture has bold/highlighting and linebreak vs documentation for hyposthesiser. See screen shot.
![SS](https://user-images.githubusercontent.com/42390382/74463808-6665dc80-4e60-11ea-8fa2-a7b09d28ec6c.PNG)

Reply:
This should be fixed now